modmesh/pilot/_svg.py

```diff
@@ -434,26 +517,37 @@ def _load_svg_file(self, filename):
         svgParser = _PathParser(filename)
         svgParser.parse()
         Epaths = svgParser.get_EPaths()
-
         sp2d = []
         cp2d = []
         for p in Epaths:
-            closedp = p.get_closed_paths()  # tuple
+            closedp = p.get_closed_paths()
             sp2d.append(closedp[0])
             cp2d.append(closedp[1])
 
         world = core.WorldFp64()
+        Point = core.Point3dFp64
 
         for i in range(len(sp2d)):
-            for s in sp2d[i]:
-                world.add_segment(s)
+            for j in range(len(sp2d[i])):
+                # the points are reflected to x-axis: (x, y) -> (x, -y)
+                world.add_segment(Point(sp2d[i].x0_at(j), -sp2d[i].y0_at(j)),
```

Comment:
Reverse y-axis values when adding segments to align with current rendering coordinate system.